### PR TITLE
Import video player

### PR DESCRIPTION
**Pull request by sigurdm**
Import the video player into the flutter/plugins repo

### CODE REVIEW

File: packages/video_player/android/gradlew [hunk 1]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```
Comment (mravn-google) [inline packages/video_player/android/gradlew]: Repo should not include the `gradlew` scripts nor `gradle-wrapper.jar` binary.

File: packages/video_player/android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        result.success(null);
+      } else if (call.method.equals("seekTo")) {
+        int location = ((Number) call.argument("location")).intValue();
+        player.seekTo(location);
+        result.success(null);
+      } else if (call.method.equals("position")) {
+        result.success(player.getPosition());
+      } else if (call.method.equals("dispose")) {
+        player.dispose();
+        result.success(null );
```
Comment (mravn-google) [inline packages/video_player/android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java]: Remove extra space.

File: packages/video_player/android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+      VideoPlayer player = videoPlayers.get(textureId);
+      if (player == null) {
+        result.error("Unknown textureId", "No video player associated with texture id " + textureId, null);
+        return;
+      }
+      if (call.method.equals("setLooping")) {
+        player.setLooping((Boolean) call.argument("looping"));
+        result.success(null);
+      } else if (call.method.equals("setVolume")) {
+        player.setVolume((Double)call.argument("volume"));
```
Comment (mravn-google) [inline packages/video_player/android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java]: Add space after cast.

File: packages/video_player/android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+        mediaPlayer.pause();
+      }
+    }
+
+    void setLooping(boolean value) {
+      mediaPlayer.setLooping(isLooping);
+    }
+
+    void setVolume(double value) {
+      float bracketedValue = (float)Math.max(0.0, Math.min(1.0, value));
```
Comment (mravn-google) [inline packages/video_player/android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java]: At space after cast (three times).

File: packages/video_player/example/android/gradlew [hunk 5]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```
Comment (mravn-google) [inline packages/video_player/example/android/gradlew]: Downloaded Gradle artifacts should be removed from here too.

File: packages/video_player/ios/Classes/VideoPlayerPlugin.m [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+  } else {
+    NSDictionary* argsMap = call.arguments;
+    int64_t textureId = ((NSNumber*) argsMap[@"textureId"]).unsignedIntegerValue;
+    VideoPlayer* player = _players[@(textureId)];
+    if ([@"dispose" isEqualToString:call.method]) {
+      [_registry unregisterTexture:textureId];
+      [_players removeObjectForKey:@(textureId)];
+      [player dispose];
+    } else if ([@"setLooping" isEqualToString:call.method]) {
+        [player setIsLooping:[argsMap objectForKey:@"looping"]];
```
Comment (mravn-google) [inline packages/video_player/ios/Classes/VideoPlayerPlugin.m]: Please adjust indentation. Here and elsewhere in ObjC code.

**Comment by mit-mit** [comment]
Please also look at the travis issues.

**Pull request merged**